### ISSUE TITLE
Sauna can decrease ACM by ~40%

### ISSUE THREAD

**Issue by sm18lr88** [geekan/HowToLiveLonger#150]
[This study](https://pubmed.ncbi.nlm.nih.gov/25705824/) followed 2,315 middle-aged men from eastern Finland for nearly 20 years, and it adjusted for many co-founders. While the demographics are limited, the length of the study, the power, and the outcomes were so significant that it has to be meaningful to every demographic:

### Key findings:

- Sauna usage of 4-7 times per week was associated with:
    - 63% decrease in sudden cardiac death
    - 48% decrease in fatal coronary heart disease
    - 50% decrease in fatal cardiovascular disease 
    - **40% decrease in all-cause mortality**: pg. 4: "Correspondent estimates for CVD mortality were 27% and 50% (P for trend < .001). Likewise, significant inverse associations were found between duration of sauna bathing and CHD (P for trend= .007) and CVD (P for trend = .03) mortality (Table 3). Frequency of sauna bathing, but not duration, was also inversely associated with all cause mortality, with a 40% reduction comparing 4 to 7 sessions vs 1 session of sauna bathing per week (Table 2 and Table 3)."

- Sauna duration >19 minutes compared to <11 minutes was associated with:
    - 52% decrease in sudden cardiac death
    - 36% decrease in fatal coronary heart disease
    - 24% decrease in fatal cardiovascular disease

- The most significant reductions were for 4-7 sauna sessions per week.

In short for the found that men who used the sauna 4-7 times per week at an average temperature of 79°C were 40% less likely to die from any cause compared to men who only used the sauna once per week over the 19 year follow-up period.

### Reference:

```
Laukkanen, T., Khan, H., Zaccardi, F., & Laukkanen, J. A. (2015). Association between sauna bathing and fatal cardiovascular and all-cause mortality events. *JAMA internal medicine*, *175*(4), 542-548. 
```
